build-system/app-index/components/proxy-form.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* eslint-disable no-unused-vars */
+
+import {Component, h, render} from 'preact';
+
+class ProxyForm extends Component {
+
+  constructor() {
+    super();
+    this.setState({
+      proxyInput: '',
+    });
```

Comment:
No need to setState from the constructor, can create the state object with the default values.

```suggestion
  constructor() {
    super();
    this.state = { proxyInput: '' };
  }
```